src/mgr/MgrClient.cc

```diff
@@ -653,9 +652,9 @@ int MgrClient::service_daemon_update_task_status(
   return 0;
 }
 
-void MgrClient::update_daemon_health(std::vector<DaemonHealthMetric>&& metrics)
+int MgrClient::update_daemon_health(std::vector<DaemonHealthMetric>&& metrics)
```

Comment:
unneeded return type change since this routine _always returns 0.

Reply:
That's there for any error reporting from the MgrClient to ServiceDaemon. We can drop it, but it's the same for `MgrClient::service_daemon_update_status`, `MgrClient::service_daemon_update_task_status` and `MgrClient::service_daemon_register`. Then those should be dropped too.